airbyte-ci/connectors/pipelines/pipelines/external_scripts/airbyte_ci_check.sh

```diff
@@ -1,14 +1,38 @@
 #!/usr/bin/env bash
 set -o errexit -o nounset -o pipefail
 
-# Check if airbyte-ci is on the PATH and pointing to the correct location
-EXPECTED_PATH="$HOME/.local/bin/airbyte-ci"
-AIRBYTE_CI_PATH=$(which airbyte-ci 2>/dev/null)
+echo "Checking if airbyte-ci is correctly installed..."
+
+INSTALL_DIR="$HOME/.local/bin"
+HUMAN_READABLE_INSTALL_DIR="\$HOME/.local/bin"
+
+# Check that the target directory is on the PATH
+# If not print an error message and exit
+if ! echo "$PATH" | grep -q "$INSTALL_DIR"; then
+    echo "The target directory $INSTALL_DIR is not on the PATH"
+    echo "Check that $HUMAN_READABLE_INSTALL_DIR is part of the PATH"
+    echo ""
+    echo "If not, please add 'export PATH=\"$HUMAN_READABLE_INSTALL_DIR:\$PATH\"' to your shell profile"
+    exit 1
+fi
 
+# Check that airbyte-ci is on the PATH
+# If not print an error message and exit
+if ! which airbyte-ci >/dev/null 2>&1; then
+    echo "airbyte-ci is not installed"
+    echo ""
+    echo "Please run 'make tools.airbyte-ci.install' to install airbyte-ci"
+    exit 1
+fi
+
+EXPECTED_PATH="$INSTALL_DIR/airbyte-ci"
+AIRBYTE_CI_PATH=$(which airbyte-ci 2>/dev/null)
```

Comment:
Super nit: we could set this `AIRBYTE_CI_PATH` variable earlier, and check whether it is empty on L21 instead of running the command twice. E.g. 

```suggestion
# Check that airbyte-ci is on the PATH
AIRBYTE_CI_PATH=$(which airbyte-ci 2>/dev/null)

# If not print an error message and exit
if [ -z "$AIRBYTE_CI_PATH" ]; then
    echo "airbyte-ci is not installed"
    echo ""
    echo "Please run 'make tools.airbyte-ci.install' to install airbyte-ci"
    exit 1
fi

EXPECTED_PATH="$INSTALL_DIR/airbyte-ci"
```